content/pl/codebase.md

```diff
@@ -0,0 +1,17 @@
+## I. Codebase
+### Jedno źródło kodu śledzone systemem kontroli wersji, wiele wdrożeń
+
+Aplikacja napisana wg dwunastu aspektów jest zawsze śledzona w systemie kontroli wersji takim jak [Git](http://git-scm.com/), [Mercurial](http://mercurial.selenic.com/), czy [Subversion](http://subversion.apache.org/). Miejsce, w którym trzymany i rewizjonowany jest kod nazywane jest *repozytorium kodu źródłowego*, często skracane do samego *code repo*, albo po prostu *repo*.
+
+*Codebase* jest więc niczym innym jak pojedynczym repo (w przypadku zcentralizowanego systemu kontroli wersji jak Subversion), albo zestawem repozytoriów, które współdzielą tzw. root commit. (w przypadku zdecentralizowanego systemu jak Git).
+
+![One codebase maps to many deploys](/images/codebase-deploys.png)
+
+Aplikacja powinna zawsze odzwierciedlać codebase:
+
+* Jeśli jest wiele źródeł, z których pobierany jest kod, nie jest to aplikacja, a system rozproszony. Każdy komponent w systemie rozproszonym jest aplikacją i każdy możte z osobna spełniać wszystkie 12 aspektów.
+* Wiele aplikacji dzielących ten sam kod jest naruszeniem 12 aspektów. Roziwązaniem tej sytuacji może być wyizolowanie współdzielonego kodu do bibliotek, które mogą być załączane przez tzw. [dependency manager](./dependencies).
```

Comment:
1. _Wiele aplikacji współdzielących kod narusza zasady **12factor app**._
2.  _Ro**ziw**ązaniem_
3. _załączane_ nie brzmi dobrze, w tym wypadku pasuje lepiej _zarządzane_, mimo że nie jest to bezpośrednie tłumaczenie z oryginału